kahuna/public/js/search/results.html

```diff
@@ -104,21 +104,9 @@
                                  ng:if="ctrl.selectionIsDeletable && ctrl.selectionCount > 0">
                 </gr-delete-image>
 
-                <a class="results-toolbar-item results-toolbar-item--right results-toolbar-item__metadata-panel-controls"
-                    ng:class="{'results-toolbar-item--disabled': !ctrl.metadataPanelAvailable, 'results-toolbar-item--active': ctrl.metadataPanelVisible}"
-                    ng:mouseover="ctrl.showMetadataPanelMouseOver()"
-                    ng:mouseleave="ctrl.showMetadataPanelMouseLeave()"
-                    ng:click="ctrl.toggleLockMetadataPanel()">
-                    <span ng:if="ctrl.metadataPanelVisible && !ctrl.metadataPanelLocked">
-                        <gr-icon-label gr-icon="lock_open">Lock info panel</gr-icon-label>
-                    </span>
-                    <span ng:if="ctrl.metadataPanelVisible && ctrl.metadataPanelLocked">
-                        <gr-icon-label gr-icon="lock">Unlock info panel</gr-icon-label>
-                    </span>
-                    <span ng:if="!ctrl.metadataPanelVisible">
-                        <gr-icon-label gr-icon="chrome_reader_mode">Show info panel</gr-icon-label>
-                    </span>
-                </a>
+                <gr-panel-button class="results-toolbar-item results-toolbar-item--right results-toolbar-item__metadata-panel-controls"
```

Comment:
Confusing that the same element is both `results-toolbar-item` and child BEM element `results-toolbar-item__metadata-panel-controls`, though it looks like it was already the case so nevermind me.